.github/workflows/ci_suite.yml

```diff
@@ -90,6 +90,9 @@ jobs:
           mysql -u root -proot -e 'CREATE DATABASE tg_ci_prefixed;'
       - name: Install rust-g
         run: |
+          sudo dpkg --add-architecture i386
+          sudo apt update || true
+          sudo apt install -o APT::Immediate-Configure=false libssl1.1:i386
           bash tools/ci/install_rust_g.sh
       - name: Compile and run tests
```

Comment:
trash?

Reply:
Yeah for some reason I had to put that in order to make a PR, I really have no idea why. Can I safely remove the file?

Reply:
As said in disc you need to revert these changes

Reply:
It seems to have worked doing it manually with your code review.